resources/js/components/Dropdown.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+    <button type="button" @click="toggle">
+        <slot />
```

Comment:
In `CharacterCount` it uses `<slot></slot>`, is there a difference?

Reply:
.vue files let you use that shorthand for closing off HTML tags. There's no difference but I changed this for consistency.